packages/components/src/Grid/Grid.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ReactNode } from 'react';
+
+import type { GapSpaceProp, HeightProp } from '@marigold/system';
+import { cn, gapSpace, height as twHeight } from '@marigold/system';
+
+import { Slot } from './GridSlot';
+
+// Helpers
+// ---------------
+export type TemplateValue =
+  | 'none'
+  | 'auto'
+  | 'min-content'
+  | 'max-content'
+  // Allow autocomplete with fixed template strings (see https://github.com/microsoft/TypeScript/issues/29729)
+  | (string & {})
+  | number;
+
+const parseGridAreas = (areas: string[]) =>
+  areas.map(area => `"${area}"`).join('\n');
+
+const parseTemplateValue = (values: TemplateValue[]) =>
+  values
```

Comment:
This should also work:

values.map(val => (typeof val === 'number' ? `${val}fr` : val)).join(' ');

Reply:
Yes, I wrote more code to make it more explicit and readable. hopefully this would help to add more logic here if needed. but I can also change it if you want to :)

Reply:
Hm okay, that always triggers me. But again, it's just a suggestion. If you think it's easier to understand that way, that's fine by me too :-)

Reply:
changed it :)